### PR TITLE
add mb44 keyboard

### PR DESCRIPTION
**Pull request by melonbred**
## Description

Add the MB-44 keyboard.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/mb44/rules.mk [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
+# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+NKRO_ENABLE = no            # USB Nkey Rollover
+BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
+RGBLIGHT_ENABLE = no        # Enable keyboard RGB underglow
+MIDI_ENABLE = no            # MIDI support
+BLUETOOTH_ENABLE = no       # Enable Bluetooth with the Adafruit EZ-Key HID
+AUDIO_ENABLE = no           # Audio output
+FAUXCLICKY_ENABLE = no      # Use buzzer to emulate clicky switches
+HD44780_ENABLE = no         # Enable support for HD44780 based LCDs
```
Comment (ridingqwerty) [inline keyboards/mb44/rules.mk:32]:
```suggestion
BLUETOOTH_ENABLE = no       # Enable Bluetooth with the Adafruit EZ-Key HID
AUDIO_ENABLE = no           # Audio output
```

File: keyboards/mb44/rules.mk [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+MCU = atmega32u2
+
+# Bootloader selection
+#   Teensy       halfkay
+#   Pro Micro    caterina
+#   Atmel DFU    atmel-dfu
+#   LUFA DFU     lufa-dfu
+#   QMK DFU      qmk-dfu
+#   ATmega32A    bootloadHID
+#   ATmega328P   USBasp
```
Comment (ridingqwerty) [inline keyboards/mb44/rules.mk:11]:
```suggestion
```

File: keyboards/mb44/readme.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+A short description of the keyboard/project
+
+* Keyboard Maintainer: [melonbred](https://github.com/melonbred)
+* Hardware Supported: The PCBs, controllers supported
+* Hardware Availability: Links to where you can find this hardware
+
+Make example for this keyboard (after setting up your build environment):
+
+    make mb44:default
+
```
Comment (ridingqwerty) [inline keyboards/mb44/readme.md:14]: Consider adding instructions on resetting the board here.

File: keyboards/babyv/keymaps/melonbred/keymap.c [hunk 4]
```diff
@@ -0,0 +1,62 @@
+/* Copyright 2020 Eye Oh! Designs
```
Comment (ridingqwerty) [inline keyboards/babyv/keymaps/melonbred/keymap.c:1]:
```suggestion
/* Copyright 2020 melonbred
```

All the other licenses specify "melonbred", I don't think it matters much but I think a consistent attribution would be preferable.

**Review by ridingqwerty — CHANGES_REQUESTED** [review]
Some minor suggestions.  The `lib/chibios` stuff will have to be removed from this PR.

**Comment by melonbred** [comment]
@ridingqwerty Thanks for your quick review! Regarding the ChibiOS stuff, is that something I need to remove? Not sure exactly what changed or why there were changes to anything ChibiOS related as this isn't an arm board.

**Comment by drashna** [comment]
You have changes to the chibiOS modules, this can happen if they wern't updated properly, etc.  

Rebasing your branch should fix the issue.

**Comment by melonbred** [comment]
> You have changes to the chibiOS modules, this can happen if they wern't updated properly, etc.
> 
> Rebasing your branch should fix the issue.

thanks, drashna! I believe the chibiOS modules have been resolved. Please let me know if I might need to do anything else for this PR.

File: keyboards/mb44/readme.md (hunk 3 above)
Comment (melonbred) [reply keyboards/mb44/readme.md]: Instructions on resetting the board have been added to readme.md.

**Review by drashna — APPROVED** [review]

File: keyboards/mb44/rules.mk [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+EXTRAKEY_ENABLE = yes       # Audio control and System control
+CONSOLE_ENABLE = no        # Console for debug
+COMMAND_ENABLE = no        # Commands for debug and configuration
+# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
+SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
+# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+NKRO_ENABLE = no            # USB Nkey Rollover
+BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
+RGBLIGHT_ENABLE = no        # Enable keyboard RGB underglow
+BLUETOOTH_ENABLE = no       # Enable Bluetooth with the Adafruit EZ-Key HID
```
Comment (fauxpark) [inline keyboards/mb44/rules.mk:21]:
```suggestion
BLUETOOTH_ENABLE = no       # Enable Bluetooth
```

File: keyboards/mb44/info.json [hunk 6]
```diff
@@ -0,0 +1,202 @@
+{
+    "keyboard_name": "MB-44",
+    "url": "",
+    "maintainer": "melonbred",
+    "width": 3,
+    "height": 2,
+    "layouts": {
+        "default": {
```
Comment (fauxpark) [inline keyboards/mb44/info.json:8]:
```suggestion
        "LAYOUT_default": {
```
etc.

**Review by fauxpark — CHANGES_REQUESTED** [review]

File: keyboards/mb44/info.json [hunk 7]
```diff
@@ -0,0 +1,202 @@
...
+                {"x":0, "y":3},
+                {"x":1, "y":3},
+                {"x":2, "y":3},
+                {"x":3, "y":3, "w":6.25},
+                {"x":9.75, "y":3},
+                {"x":10.75, "y":3},
+                {"x":11.75, "y":3}
+            ]
+        },
+        "2u_split": {
```
Comment (fauxpark) [inline keyboards/mb44/info.json:55]: etc.

Comment (melonbred) [reply keyboards/mb44/info.json]: `LAYOUT_` added to all layouts. Sorry about that!

**Review by fauxpark — APPROVED** [review]

**Pull request merged**